diff --git a/lives-plugins/marcos-encoders/lives_avi_encoder3 b/lives-plugins/marcos-encoders/lives_avi_encoder3
index be0f4e3..fc3a002 100644
--- a/lives-plugins/marcos-encoders/lives_avi_encoder3
+++ b/lives-plugins/marcos-encoders/lives_avi_encoder3
@@ -27,7 +27,7 @@ See my vids at http://amv.reimeika.ca !
 
 Copyright (C) 2008 - 2018 Salsaman (salsaman+lives@gmail.com)
 """
-
+import os
 version = '0.0.19'
 if os.name == 'nt':
     convert = 'mgkvert'
diff --git a/lives-plugins/plugins/encoders/multi_encoder3 b/lives-plugins/plugins/encoders/multi_encoder3
index f4e4a77..9a78c93 100644
--- a/lives-plugins/plugins/encoders/multi_encoder3
+++ b/lives-plugins/plugins/encoders/multi_encoder3
@@ -189,7 +189,7 @@ def is_installed(prog):
                 std_in.close()
         
 
-        if 'command not found' in std_out_err.read():
+        if b'command not found' in std_out_err.read():
             if verbose:
                 print(prog + ': missing dependencies', file=sys.stderr)
             found = 0
